README.md

```diff
@@ -35,9 +35,11 @@ to find out more about the origin story and design philosophy of Envoy.
 * [envoy-maintainers](https://groups.google.com/forum/#!forum/envoy-maintainers): Use this list
   to reach all core Envoy maintainers.
 * [Twitter](https://twitter.com/EnvoyProxy/): Follow along on Twitter!
-* [Slack](https://envoyproxy.slack.com/): Slack, to get invited go here: http://envoyslack.cncf.io (NOTE: Response to user
-  questions is best effort on Slack. For a "guaranteed" response please email envoy-users@ per the
-  guidance in the following linked thread.).
+* [Slack](https://envoyproxy.slack.com/): Slack, to get invited go [here](http://envoyslack.cncf.io). 
+  We have the IRC/XMPP gateways enabled if you prefer either of those. Once an account is created,
+  connection instructions for IRC/XMPP can be found [here](https://envoyproxy.slack.com/account/gateways).
+  * (NOTE: Response to user questions is best effort on Slack. For a "guaranteed" response please email 
```

Comment:
Formatting with bullet point and parenths is a bit weird in preview..

Reply:
Yeah I just fixed that.